### DISCUSSION TITLE
Missing disk labels in Storage

### DISCUSSION THREAD

**Discussion by pythongates** [cockpit-project/cockpit#17662] [Q&A]
Disk label column was super helpful in the Storage but latest version removed it.  Any way to get it back?  Thanks!

**Comment by marusak** [comment]
Sorry I am not sure what exactly you mean. Could you please post some pictures and mention from which version you updated to which version? Thanks!

**Reply by pythongates** [reply]
Sorry I should have been a bit more detailed.
Here is what the view looks like from the default version Ubuntu 20.04 installs which v215
Notice the label is visible.
![Cockpit-Version-215](https://user-images.githubusercontent.com/25665553/185985623-b24758d1-385e-4506-b214-a6f186088de2.JPG)

Here is the same system after updating to v264.
Notice the label is replaced with the dev.  Is there a way to revert back to showing the disk label (if one exists) as the default?
![Cockpit-Version-264-1~bpo20 04 1](https://user-images.githubusercontent.com/25665553/185985659-7ef70836-b8c9-4a6b-8581-bbd889918a87.JPG)

Thank you for taking the time to review my Q&A.

**Comment by mvollmer** [comment]
This was changed in Cockpit in #16518 (cce897d5395b6dbface8444b35435a196507b0af), for no apparent reason... I guess we can put the label back: #17668.

**Reply by pythongates** [reply]
Or may I suggest both?  Keep as is and add a new label column?  That would be even better.
Source, Label, Mount, Size
I work with hundreds (>300) of disks attached to servers.  Having this type of info in a quick web view is invaluable.  
Thank you!
